collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/GuidanceTest.kt

```diff
@@ -0,0 +1,56 @@
+package org.odk.collect.android.feature.formentry
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.odk.collect.android.R
+import org.odk.collect.android.support.pages.MainMenuPage
+import org.odk.collect.android.support.pages.ProjectSettingsPage
+import org.odk.collect.android.support.rules.CollectTestRule
+import org.odk.collect.android.support.rules.TestRuleChain.chain
+
+@RunWith(AndroidJUnit4::class)
+class GuidanceTest {
+    private var rule = CollectTestRule()
+
+    @get:Rule
+    var copyFormChain: RuleChain = chain().around(rule)
+
+    @Test
+    fun guidanceForQuestion_ShouldDisplayAlways() {
```

Comment:
I think both these tests could use improvements in their names. This one should have something like `whenGuidanceSetToAlwaysShow...`.